pkg/assets/apps.go

```diff
@@ -100,14 +100,14 @@ func applyApps(apps []string, applier readerApplier, render RenderFunc, params R
 	defer lock.Unlock()
 
 	for _, app := range apps {
-		logrus.Infof("applying apps api %s", app)
+		klog.InfoS("Applying apps api", "apps", app)
 		objBytes, err := appsassets.Asset(app)
 		if err != nil {
 			return fmt.Errorf("error getting asset %s: %v", app, err)
 		}
 		applier.Reader(objBytes, render, params)
 		if err := applier.Applier(); err != nil {
-			logrus.Warningf("failed to apply apps api %s: %v", app, err)
+			klog.InfoS("Failed to apply apps api", "apps", app, "err", err)
```

Comment:
We are changing the level of the log message (from "W" warning to "I" info) , I suggest we keep using the klog.Infof  /Warningf / Errorf interfaces.

Would that make sense?

@oglok @rootfs

Reply:
Ack, after reading, if that's the future of k8s I believe it makes sense @husky-parul.

I'm going to make another pass checking the context of the log messages, in some cases, may be Warnings need to become Errors for higher severity, since they seem to have dropped Warning completely out of structure logging in klog.